sdk/lib/3rdparty/freetype/src/smooth/ftgrays.c

```diff
@@ -1878,12 +1811,25 @@ typedef ptrdiff_t  FT_PtrDist;
            outline->contours[outline->n_contours - 1] + 1 )
       return FT_THROW( Invalid_Outline );
 
+#if !defined(FT_STATIC_RASTER) && defined(__REACTOS__)
+    worker = malloc(sizeof(*worker));
+    if (!worker)
+      return FT_THROW(Out_Of_Memory);
+#endif /* !defined(FT_STATIC_RASTER) && defined(__REACTOS__) */
+
     ras.outline = *outline;
 
     if ( params->flags & FT_RASTER_FLAG_DIRECT )
     {
       if ( !params->gray_spans )
+#ifdef __REACTOS__
```

Comment:
In theory, it seems that all these `free(worker)` ROS-specific additions in this function, should be surrounded by:
`#if !defined(FT_STATIC_RASTER) && defined(__REACTOS__)` instead.